dataactvalidator/config/sqlrules/sqlRules.csv

```diff
@@ -156,4 +156,6 @@ D41,"For PrimaryPlaceOfPerformanceCode XX##### city must exist in provided state
 D41,"When provided, PrimaryPlaceofPerformanceZIP+4 must be in the state specified by PrimaryPlaceOfPerformanceCode.","When provided, PrimaryPlaceofPerformanceZIP+4 must be in the state specified by PrimaryPlaceOfPerformanceCode.",n,detached_award,fatal,,d41_detached_award_financial_assistance_3
 D42,"PrimaryPlaceOfPerformanceForeignLocationDescription is required for foreign places of performance (i.e., when PrimaryPlaceOfPerformanceCountryCode does not equal USA).","PrimaryPlaceOfPerformanceForeignLocationDescription is required for foreign places of performance (i.e., when PrimaryPlaceOfPerformanceCountryCode does not equal USA).",n,detached_award,fatal,,d42_detached_award_financial_assistance_1
 D42,"PrimaryPlaceOfPerformanceForeignLocationDescription must be blank for domestic recipients (i.e., when PrimaryPlaceOfPerformanceCountryCode = USA).","PrimaryPlaceOfPerformanceForeignLocationDescription must be blank for domestic recipients (i.e., when PrimaryPlaceOfPerformanceCountryCode = USA).",n,detached_award,fatal,,d42_detached_award_financial_assistance_2
+D43,"When primaryplaceofperformancecountrycode is not USA Congressional District must be empty","primaryplaceofperformancecongressionaldistrict must be blank when primaryplaceofperformancecountrycode is not USA",n,detached_award,fatal,,d43_detached_award_financial_assistance_1
+D43,"PrimaryPlaceOfPerformanceCongressionalDistrict must be provided and valid when PrimaryPlaceofPerformanceZIP+4 is provided","PrimaryPlaceOfPerformanceCongressionalDistrict must be provided and valid when PrimaryPlaceofPerformanceZIP+4 is provided",n,detached_award,fatal,,d43_detached_award_financial_assistance_2

```

Comment:
This is not accurate, PrimaryPlaceOfPerformanceCongressionalDistrict is NOT required when PrimaryPlaceofPerformanceZIP+4 is provided.